src/ucp/core/ucp_rkey.inl

```diff
@@ -11,6 +11,23 @@
 #include "ucp_worker.h"
 
 
+static UCS_F_ALWAYS_INLINE khint_t
+ucp_rkey_config_hash_func(ucp_rkey_config_key_t rkey_config_key)
+{
+    return (khint_t)rkey_config_key.md_map ^
+           (rkey_config_key.ep_cfg_index << 8) ^
+           (rkey_config_key.mem_type << 16);
+}
+
+static UCS_F_ALWAYS_INLINE int
+ucp_rkey_config_is_equal(ucp_rkey_config_key_t rkey_config_key1,
+                                ucp_rkey_config_key_t rkey_config_key2)
```

Comment:
indentation looks strange. is this Clang format?

Reply:
fixed